pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -351,6 +351,28 @@ Answer.Picklist:
           type: array
           items:
             $ref: '../pepper.yml#/components/schemas/SelectedOption'
+Answer.File:
+  allOf:
+    - $ref: '../pepper.yml#/components/schemas/Answer'
+    - type: object
+      required:
+        - value
+      properties:
+        value:
+          type: object
+          $ref: '../pepper.yml#/components/schemas/FileInfo'
+FileInfo:
```

Comment:
This should include the `uploadId` as well, if one exists.

Reply:
Marking this resolved- walked through a couple uploads and there doesn't seem to be a reason for the client to hold onto the upload id once it's been `PATCH`ed.